src/utils/components/ExportModal/ShowProgress.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { FC } from 'react';
+
+import { IoK8sApiCoreV1Pod } from '@kubevirt-ui/kubevirt-api/kubernetes';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { isEmpty } from '@kubevirt-utils/utils/utils';
+import {
+  Alert,
+  AlertVariant,
+  ProgressStep,
+  ProgressStepper,
+  ProgressStepVariant,
+  Stack,
+} from '@patternfly/react-core';
+
+import { STATUS_TO_PROGRESS_VARIANT, UPLOAD_STATUSES } from './constants';
+import { exportFailed, exportInProgress } from './utils';
+
+type ShowProgressProps = {
+  uploadPod: IoK8sApiCoreV1Pod;
+};
+
+const ShowProgress: FC<ShowProgressProps> = ({ uploadPod }) => {
+  const { t } = useKubevirtTranslation();
+
+  const phase = uploadPod?.status?.phase;
+
```

Comment:
nitpick: Are the empty lines between these variable assignments necessary? They seem related to one another.